public/index.html

```diff
@@ -2,19 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+  
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+  
+ 
+  
```

Comment:
Please, remove empty lines

Reply:
The empty lines are still not removed. Don't re-request review if the comments from the previous review aren't addressed

Reply:
Well, i will creat new work branch?

Reply:
Yes, create another branch and move these changes from this branch to the new one

Reply:
what is correct name to choose?

Reply:
The branch name should concisely describe the change it introduces; see:
- https://codingsight.com/git-branching-naming-convention-best-practices/
- https://stackoverflow.com/questions/273695/what-are-some-examples-of-commonly-used-practices-for-naming-git-branches